changelog/deprecate_class_allocators.dd

```diff
@@ -0,0 +1,20 @@
+Class allocators and deallocators have been deprecated
+
+Class allocators and deallocators have been planned for deprecation for years.  Starting with this release the following code will emit deprecation messages.
+
+---
+class Foo
+{
+    new(size_t size)         // deprecation message
+    {
+        return malloc(size);
+    }
+
+    delete(void* obj)        // deprecation message
+    {
+        free(obj);
+    }
+}
+---
+
+For more information see https://dlang.org/deprecate.html#Class%20allocators%20and%20deallocators
```

Comment:
Probably a good idea to point users to alternatives like std.experimental.allocator and other RAII articles.
See also: https://issues.dlang.org/show_bug.cgi?id=18613

Reply:
Please enumerate specifically which articles I should included and I'll add them.

Reply:
Probably this page is the best we have at the moment - https://dlang.org/blog/the-gc-series

It doesn't really mention stdx.allocator, so I would probably mentioned that separately.

How about:

> Many alternatives for class allocators/deallcators exist, among many is the generic $(REF make, std,experimental,allocator) and $(REF dispose, std,experimental,allocator) from the allocator package. For other alternatives, see $(LINK2 https://dlang.org/blog/the-gc-series, the recent article about memory allocation on the DBlog)

(just an idea)

Reply:
Done.  Thanks!